index.html

```diff
@@ -15,16 +15,21 @@
         <h1><span>K</span>-Kal</h1>
       </a>
     </div>
+    <button id="open" class="open-menu" ><i class="fa-solid fa-bars"></i></button>
     <nav class="nav">
-      <div class="nav-container">
-        <ul class="side-bar" id="menu">
-          <li><a href="#portfolio">Portfolio</a></li>
-          <li><a href="#about">About</a></li>
-          <li><a href="#">Contact</a></li>
-        </ul>
-        <a href="#menu" class="nav-menu"><i class="fa-solid fa-bars nav-icon"></i></a>
-        <a href="#" class="nav-menu nav-menu-second"><i class="fa-solid fa-x nav-icon"></i></a>
-      </div>
+      <button id="close" class="close-menu"><i class="fa-solid fa-x"></i></button>
```

Comment:
- [ ] Kindly consider fixing the position of this close-menu button. it should be exactly identical to the figma design. you can fix it using margin property.
- [ ] Also kindly consider fixing the background color. the background color of the mobile-menu is a bit dull in your design. consider making it identical to the figma design.
your design
![s](https://github.com/Kal2403/my-portfolio-project/assets/96683843/5cbc7c49-de93-4886-b809-3e8c4e7a349d)

figma design
 
![fd](https://github.com/Kal2403/my-portfolio-project/assets/96683843/316d990e-a34f-433a-b7db-ecc50f61d351)